src/ontology/imports/other_import.owl

```diff
@@ -51,6 +57,41 @@ SubClassOf(<http://purl.obolibrary.org/obo/OGMS_0000031> <http://purl.obolibrary
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/UPHENO_0001001> "phenotype"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/UPHENO_0001001> <http://purl.obolibrary.org/obo/PATO_0000001>)
 
+# Class: <http://purl.obolibrary.org/obo/OBI_0000967> (container)
```

Comment:
There's an OBI import now. Please add the CURIE ids to the `src/ontology/imports/obi_terms.txt` and run `sh run.sh make imports/obi_import.owl`. Let me know if you need help, or I can do it.

Reply:
@jmwhorton have you had a chance to review @anitacaron requested change?

Reply:
Apologies, it's been a busy few weeks.  I should be able to take care of it this week.  Appreciate you checking in on it.

Reply:
Please make sure to pull the changes in your local branch version. 😄

Reply:
The obi terms have been moved, and the `make` command has been run.

Reply:
Thank you, @jmwhorton!